server/branch/permissions.js

```diff
@@ -1,6 +1,33 @@
 import { Branch, BranchPermission, PubManager, CommunityAdmin } from '../models';
 import { checkIfSuperAdmin } from '../utils';
-import calculateBranchAccess from './calculateBranchAccess';
```

Comment:
Just for clarity, is this and the related diffs below simply refactoring `calculateBranchAccess` into `/permissions` and renaming it `getBranchAccess`? I'm supportive of that, just want to make sure I'm not missing any logic updates.

Reply:
Yep, that's the only thing!